css/_christmas.scss

```diff
@@ -0,0 +1 @@
+$color-white: #F5F4D7;
```

Comment:
jakas super ambitna zmiana styli to to nie jest :P ale powiedzmy ze poprawnie podpiales style thema

Reply:
W tym przykładzie co wysłałeś na slacku było coś poza kolorem tła? (Nie licząc jeszcze choinki)